app/models/equipment.rb

```diff
@@ -1,5 +1,13 @@
 class Equipment < ApplicationRecord
+  has_one_attached :image
```

Comment:
It's going to be just one image or many images?

Reply:
We checked the design again and it's just one image @jose-Abel

Reply:
Ok cool, thanks, so is approved!

Reply:
Great! Thanks